target/linux/mediatek/image/filogic.mk

```
@@ -889,6 +889,19 @@ define Device/glinet_gl-mt2500
 endef
 TARGET_DEVICES += glinet_gl-mt2500
 
+define Device/glinet_gl-mt2500-airoha
+  DEVICE_VENDOR := GL.iNet
+  DEVICE_MODEL := GL-MT2500
+  DEVICE_DTS := mt7981b-glinet-gl-mt2500-v2
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS_LOADADDR := 0x47000000
+  DEVICE_PACKAGES := -wpad-basic-mbedtls e2fsprogs f2fsck mkf2fs kmod-usb3 kmod-phy-airoha-en8811h airoha-en8811h-firmware
+  SUPPORTED_DEVICES += glinet,mt2500-emmc
```

As far as I understood in the comments it is not possible to detect from the outside which PHY is in the device. 
I think we should add the other as supported device too:
```suggestion
  SUPPORTED_DEVICES += glinet,mt2500-emmc glinet,gl-mt2500
```
and same for the other target. 

This will allow users to easily switch between them.

---

I added `glinet,gl-mt2500` as a supported device to `glinet,gl-mt2500-airoha` and `glinet,gl-mt2500-airoha` to `glinet,gl-mt2500` so they can be easily switched between. Thank you for the review!